prost 0.3 update breaks Conduit generated code

After updating [my Conduit branch that depends on `tower-grpc` from this repo](https://github.com/runconduit/conduit/pull/225) to depend on `prost` 0.3.0, I see the following compile error:

```
   Compiling conduit-proxy v0.1.3 (file:///Users/eliza/Code/go/src/github.com/runconduit/conduit/proxy)
error[E0432]: unresolved import `super::Update`
   --> /Users/eliza/Code/go/src/github.com/runconduit/conduit/target/debug/build/conduit-proxy-e25c6e4edf2849f0/out/conduit.proxy.telemetry.rs:147:17
    |
147 |     use super::{Update, ReportRequest, ReportResponse};
    |                 ^^^^^^ no `Update` in `control::pb::proxy::telemetry`

error: aborting due to previous error

error: Could not compile `conduit-proxy`.

```

--- a/.travis.yml
+++ b/.travis.yml
@@ -14,6 +14,9 @@ matrix:
 
 script:
   - cargo test
+  # build test crates
+  - cargo build -p multifile
+  - cargo build -p collide
   # run interop tests
   - ./tower-grpc-interop/travis-interop.sh
 

--- a/tower-grpc-build/src/client.rs
+++ b/tower-grpc-build/src/client.rs
@@ -1,6 +1,5 @@
 use codegen;
 use prost_build;
-use std::fmt;
 
 /// Generates service code
 pub struct ServiceGenerator;

--- a/tower-grpc-build/src/lib.rs
+++ b/tower-grpc-build/src/lib.rs
@@ -109,6 +109,10 @@ impl prost_build::ServiceGenerator for ServiceGenerator {
         self.root_scope
             .fmt(&mut fmt)
             .expect("formatting root scope failed!");
+        // Reset the root scope so that the service generator is ready to
+        // generate another file. this prevents the code generated for *this*
+        // file being present in the next file.
+        self.root_scope = codegen::Scope::new();
     }
 }
 

--- a/tower-grpc-build/src/server.rs
+++ b/tower-grpc-build/src/server.rs
@@ -1,8 +1,6 @@
 use codegen;
 use prost_build;
 
-use std::fmt;
-
 /// Generates service code
 pub struct ServiceGenerator;
 

